bindata/network/multus-networkpolicy/multus-networkpolicy.yaml

```diff
@@ -0,0 +1,70 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: multus-networkpolicy
+  namespace: openshift-multus
+  labels:
+    app: multus-networkpolicy
+    namespace: openshift-multus
+  annotations:
+    kubernetes.io/description: |
+      This DaemonSet launches the Multus networkpolicy component on each node.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+    networkoperator.openshift.io/non-critical: ""
```

Comment:
This could be considered critical, in a sense. Will it delay initial cluster spinup?

Reply:
Got it. Changed.